cuda/runtime/BUILD.local_cuda

```diff
@@ -317,6 +317,31 @@ cc_library(
     ]),
 )
 
+# NVRTC
+cc_import(
```

Comment:
Please use `cc_import_versioned_sos`, see [113](https://togithub.com/bazel-contrib/rules_cuda/issues/113) for more details.